### PR TITLE
create form upload image

### PR DESCRIPTION
**Pull request by chungdaotn**

### CODE REVIEW

File: B-CLIENT/gotrip/src/auth/upload-image/UploadImage.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                        placeholder="File name:"
+                        readonly>
+                    </input>
+                </div>
+                <div className="upload-image-container__form-title mt-3 d-flex justify-content-center">
+                    <span>
+                        The image uploaded will be rendered in side the box below.
+                    </span>
+                </div>
+                <div className="upload-image-container__form-image">
```
Comment (dnhieu1992) [inline B-CLIENT/gotrip/src/auth/upload-image/UploadImage.jsx:25]: rename to upload-image-container__image-preview

File: B-CLIENT/gotrip/src/auth/upload-image/UploadImage.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+const UploadImage = () => {
```
Comment (dnhieu1992) [inline B-CLIENT/gotrip/src/auth/upload-image/UploadImage.jsx:1]: Move to shared/components/forrms

File: B-CLIENT/gotrip/src/shared/sass/upload-image/upload-image.scss [hunk 3]
```diff
@@ -0,0 +1,49 @@
+.upload-image-container{
+    width: 100%;
+    height: 100vh;
```
Comment (dnhieu1992) [inline B-CLIENT/gotrip/src/shared/sass/upload-image/upload-image.scss:3]: height phụ thuộc vào parent element, không set vh

File: B-CLIENT/gotrip/src/auth/upload-image/UploadImage.jsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        <div className="upload-image-container">
+            <h1 className="upload-image-container__header">Upload image</h1>
+            <div className="upload-image-container__form ">
+                <div className="upload-image-container__form-button">
+                    <span className="btn btn-default btn-file">
+                        <i className="fa fa-cloud-upload"></i> &nbsp;
+                        Choose File <input type="file" id="imgInp" />
+                    </span>
+                </div>
+                <div className="upload-image-container__form-element">
```
Comment (dnhieu1992) [inline B-CLIENT/gotrip/src/auth/upload-image/UploadImage.jsx:12]: No need input tag

File: B-CLIENT/gotrip/src/auth/upload-image/UploadImage.jsx (hunk 1 above)
Comment (chungdaotn) [reply B-CLIENT/gotrip/src/auth/upload-image/UploadImage.jsx:25]: done

File: B-CLIENT/gotrip/src/auth/upload-image/UploadImage.jsx (hunk 2 above)
Comment (chungdaotn) [reply B-CLIENT/gotrip/src/auth/upload-image/UploadImage.jsx:1]: done

File: B-CLIENT/gotrip/src/shared/sass/upload-image/upload-image.scss (hunk 3 above)
Comment (chungdaotn) [reply B-CLIENT/gotrip/src/shared/sass/upload-image/upload-image.scss:3]: done

File: B-CLIENT/gotrip/src/auth/upload-image/UploadImage.jsx (hunk 4 above)
Comment (chungdaotn) [reply B-CLIENT/gotrip/src/auth/upload-image/UploadImage.jsx:12]: done

**Pull request merged**